transients_nearby_galaxies/transients_nearby_galaxies.ipynb

```diff
@@ -0,0 +1,1182 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">sectors['sequence_number']
</pre><p>should also retrieve this</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>